util/src/main/java/io/kubernetes/client/util/WebSockets.java

```diff
@@ -0,0 +1,107 @@
+package io.kubernetes.client.util;
+
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.ApiException;
+import io.kubernetes.client.Pair;
+
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.Response;
+import com.squareup.okhttp.ResponseBody;
+import com.squareup.okhttp.ws.WebSocket;
+import com.squareup.okhttp.ws.WebSocketListener;
+
+import static com.squareup.okhttp.ws.WebSocket.BINARY;
+import static com.squareup.okhttp.ws.WebSocket.TEXT;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import okio.Buffer;
+
+public class WebSockets {
+    /**
+     * A simple interface for a listener on a web socket
+     */
+    public interface SocketListener {
```

Comment:
Yeah, I did this in a later PR, see:

https://github.com/kubernetes-client/java/pull/24/files#diff-72ca04ac95ffb12a152f92ed801c4b5a